canonical-web-and-design/jaas-dashboard
# Offline images

## Done

The dashboard could be used in a restricted environment so we shouldn't use the assets server for static assets.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Ensure all images and icons loads as expected

## Details

Fixes #101
